### DISCUSSION TITLE
[Feature Request] Direct audio input/output

### DISCUSSION THREAD

**Discussion by MushPsykotik** [apohl79/audiogridder#969] [Ideas]
Hi,

I really enjoy your tool and i think it could be useful to be able to host a VST on the server but it receive a direct input for instruments to reduce input delay time and/or send audio output on server side to use with a external mixer or like a "real" dsp soundcard for example, still to reduce delay time.

Thanks for all your work, it make a great studio !

**Comment by kwalker99** [comment]
I came here to ask the same question... The overhead of running really lower buffer sizes in AG plugin on a mac in logic (this is no fault of AG, but likely due to the network DMA overhead in macos for certain TB network adapters), is really high in comparison to just feeding audio between two machines via RME MADI at 1.5ms, if we had the option to leverage Audio I/O on the server side it could open up some amazing possibilities for remote controlling VST instruments feeding out into analog or digital mixers or even back in to another DAW.. +1 from me on this feature request.

**Comment by apohl79** [comment]
Feature request created. Cheers!
